Utilities/DataFlow/CMakeLists.txt

```diff
@@ -29,24 +34,31 @@ set(LIBRARY_NAME ${MODULE_NAME})
 set(BUCKET_NAME ${MODULE_BUCKET_NAME})
 
 O2_GENERATE_LIBRARY()
+target_compile_options(${LIBRARY_NAME} PUBLIC -O0 -g)
```

Comment:
Hm it is not clear to me what you want to do. Do you want to compile tests only with the debug options (`-O0 -g`), independently from the global ones?